src/file.cr

```diff
@@ -694,6 +694,11 @@ class File < IO::FileDescriptor
     Path.new(first, *parts).to_s
   end
 
+  # :ditto:
+  def self.join(first : String | Path) : String
```

Comment:
Does it work if you change the method above to just be `def self.join(*parts : String | Path) : String` ?

Reply:
Yes! 😸